site/en/home/home.md

```diff
@@ -87,15 +87,15 @@ id: home.md
 
 ## What's new in docs
 
-_Apr 2022_
-
-- Added Milvus cluster installation guidance with [Ansible](install_cluster-ansible.md).
-- Remade API Reference of [PyMilvus](https://milvus.io/api-reference/pymilvus/v2.0.2/About.md) and [Milvus GO SDK](https://milvus.io/api-reference/go/v2.0.0/About.md).
-
-_Mar 2022_
-
-- Added guidance on [distance calculation](calculate_distance.md).
-- Added documentation on how to [load](load_partition.md) and [release](release_partition.md) a partition.
-- Added Milvus [benchmark](benchmark.md) report.
+_Jul 2022_
+
+- Added guidance on how to [configure Milvus with Milvus Operator](configure_operator.md).
+- Added guidance on how to [configure object storage with Docker Compose or Helm](deploy_s3.md).
```

Comment:
For line 93- 96, what about "Added guidance on how to configure [object storage](deploy_s3.md), [meta storage](deploy_etcd.md), and [message storage](deploy_pulsar.md) in Milvus with Docker Compose or Helm."?